scripts/validate.sh

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
 
 # usage: ./validate.sh bugid.bug
-# 
-# install: https://github.com/any-json/any-json 
-# install: https://github.com/jessedc/ajv-cli
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-SCHEMA="${DIR}/robust.json"
 echo Checking schema compliance for $1
-any-json --input-format=yaml $1 | ajv --verbose --errors=text -s ${SCHEMA} -d /dev/stdin 2>&1 >/dev/null | sed "s/\/dev\/stdin/File $1/"
+python3 yamale_validator.py $1
```

Comment:
Update to allow script to be executed from outside of the `scripts` directory.

Reply:
Should we go "all out" and add `argparse` support?

Then add an arg to specify the location of the schema.